Lib/queue.py

```diff
@@ -8,16 +8,26 @@
 from heapq import heappush, heappop
 from time import monotonic as time
 
+try:
```

Comment:
The usual way to incorporate the C extension is to put a "from _queue import SimpleQueue"